gradle-plugin/src/main/kotlin/org/mozilla/android/MozillaPlugin.kt

```diff
@@ -6,9 +6,34 @@ package org.mozilla.android
 
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.artifacts.repositories.MavenArtifactRepository
+import java.net.URI
 
 class MozillaPlugin : Plugin<Project> {
 
     override fun apply(project: Project) {
+        with(project) {
+            // Add a custom repository for application-services dependency resolution.
+            // This should be temporary while https://github.com/mozilla/application-services
+            // isn't publishing to maven.mozilla.org yet.
+            // See https://github.com/mozilla/application-services/issues/252.
+            val customName = "appservices"
+            val customURI = URI.create("https://dl.bintray.com/ncalexander/application-services")
+
+            // If there's already a Maven repo with the right URL, or even the right name, roll with it.
+            // The name gives the opportunity to customize, if it helps in the wild.
+            val existing = project.repositories.find {
+                (it is MavenArtifactRepository) && (it.url == customURI || it.name == customName)
+            }
+
+            // Otherwise, inject the dependency.
+            if (existing == null) {
+                logger.info("adding '${customName}' Maven repository with url '${customURI.toASCIIString()}'")
```

Comment:
Maybe include the project name in this string?  Not sure if it'll be clear from logging metadata.

Reply:
Logger is scoped to the project (that wasn't obvious to me at first, since I forgot what `with` did) so I'm _assuming_ that logs will be helpful.

Reply:
They're semi-useful currently. I'll update.